usaspending_api/etl/es_etl_helpers.py

```diff
@@ -185,10 +186,13 @@ def execute_sql_statement(cmd, results=False, verbose=False):
     rows = None
     if verbose:
         print(cmd)
-    with connection.cursor() as cursor:
-        cursor.execute(cmd)
-        if results:
-            rows = db_rows_to_dict(cursor)
+
+    with psycopg2.connect(dsn=get_database_dsn_string()) as connection:
+        connection.autocommit = True
```

Comment:
What does this do?

Reply:
http://initd.org/psycopg/docs/usage.html#transactions-control